MentorsManagement.API/Program.cs

```diff
@@ -10,12 +11,24 @@ private static void Main(string[] args)
         var builder = WebApplication.CreateBuilder(args);
 
         // Add services to the container.
-        builder.Services.AddDbContext<MentorDbContext>(options =>
-            options.UseSqlServer(builder.Configuration.GetConnectionString("DBConnection")));
+
+        //Mongo Db config
+        builder.Services.Configure<MongoDbSettings>(builder.Configuration.GetSection("MongoDbSettings"));
+
+        MongoDbSettings mongoDbSettings = builder.Configuration.GetSection("MongoDbSettings").Get<MongoDbSettings>();
+
+        builder.Services.AddSingleton<IMongoClient, MongoClient>(sp =>
+        {
+            return new MongoClient(mongoDbSettings?.ConnectionString);
```

Comment:
no need of return and method. 
use lambda

builder.Services.AddSingleton<IMongoClient, MongoClient>(_ =>MongoClient(mongoDbSettings?.ConnectionString))